docs/gl_objects/personal_access_tokens.rst

```diff
@@ -52,6 +52,13 @@ Revoke the personal access token currently used::
 
     gl.personal_access_tokens.delete("self")
 
+Rotate a personal access token and retrieve its new value::
+
+    gl.personal_access_tokens.rotate(42)
```

Comment:
isn't that a bit confusing.

In my mind it should be `gl.personal_access_tokens.get(42, lazy=True).rotate()`

Reply:
Done by switching the docs to show the object method first.